sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalogSuite.scala

```diff
@@ -937,10 +985,22 @@ class SessionCatalogSuite extends PlanTest {
 
   test("list partitions with invalid partial partition spec") {
```

Comment:
The above one is verifying the `catalog.listPartitionNames` and this one is verifying `catalog.listPartitions`. Should we keep them separate?